packages/hooks/src/configure.ts

```diff
@@ -0,0 +1,15 @@
+export type ErrorNotifierFunction = (message: string, error: unknown) => void;
```

Comment:
I'm thinking a bit about this configure piece here and how it fits into the greater systems in Atlantis. No direct comments now but more a heads up that I may have a followup or questions here. 😄